src/lib.rs

```diff
@@ -516,7 +516,7 @@ pub fn generate(xml: &str, config: &Config) -> Result<Generation> {
     let device = svd::parse(xml)?;
     let mut device_x = String::new();
     let items =
-        generate::device::render(&device, &config, &mut device_x).or(Err(SvdError::Render))?;
+        generate::device::render(&device, &config, &mut device_x, ".").or(Err(SvdError::Render))?;
```

Comment:
shouldn't this be configurable for the lib as well?

Reply:
Added to config